migrations/20141201172522_alter_user_column_defaults.php

```diff
@@ -19,8 +19,8 @@ public function up()
             ->changeColumn('reset_password_code', 'string', ['null' => true])
             ->changeColumn('first_name', 'string', ['null' => true])
             ->changeColumn('last_name', 'string', ['null' => true])
-            ->changeColumn('created_at', 'datetime', ['default' => '0000-00-00 00:00:00'])
-            ->changeColumn('updated_at', 'datetime', ['default' => '0000-00-00 00:00:00'])
+            ->changeColumn('created_at', 'datetime', ['default' => '1000-01-01 00:00:00'])
```

Comment:
@mdwheele 

To be honest, I'm not sure how much sense a default value like that actually makes, how about defaulting to `NULL` instead?